coverage_comment/coverage.py

```diff
@@ -82,10 +85,20 @@ class DiffCoverage:
     files: dict[pathlib.Path, FileDiffCoverage]
 
 
-def compute_coverage(num_covered: int, num_total: int) -> decimal.Decimal:
-    if num_total == 0:
+def compute_coverage(
+    num_covered: int,
+    num_total: int,
+    num_branches_covered: int = 0,
+    num_branches_total: int = 0,
+) -> decimal.Decimal:
+    """Compute the coverage percentage, with or without branch coverage."""
+    num_branches_covered = cast(int, num_branches_covered)
+    num_branches_total = cast(int, num_branches_total)
+    numerator = decimal.Decimal(num_covered + num_branches_covered)
+    denominator = decimal.Decimal(num_total + num_branches_total)
```

Comment:
I decided to modify `compute_coverage` so that branch data is always included in the computations. The numbers now default to `0` (instead of `None`) if branch data isn't being tracked, so the math comes out right no matter what. If branch data is collected, it's factored in to the computations. If not, then `0` gets added to various values, and the numbers come out exactly the same either way.